### PR TITLE
add Odigos icon

### PR DESCRIPTION
**Pull request by BenElferink**
odigos

**Issue:** n/a

**Popularity metric:**

[Website](https://odigos.io)
[GitHub](https://github.com/odigos-io/odigos)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I work at Odigos

### CODE REVIEW

**Comment by LitoMore** [comment]
@BenElferink Sorry, the Similarweb rank and GitHub stars do not meet our requirements.

Do you have any other popular metrics to prove its popularity?

If not, you can return here once your project reaches 5000+ GitHub stars.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11253,6 +11253,16 @@
...
         },
+        {
+            "title": "Odigos",
+            "hex": "F9F9F9",
+            "source": "https://odigos.io/icons/brand",
+            "aliases": {
+                "aka": [
+                    "keyval"
+                ]
+            }
```
Comment (LitoMore) [inline _data/simple-icons.json:11264]: Sorry, I don't think `keyval` is a well-known aliases name for this project.

Comment (BenElferink) [reply _data/simple-icons.json]: OK, I removed it 👍

**Comment by BenElferink** [comment]
> @BenElferink Sorry, the Similarweb rank and GitHub stars do not meet our requirements.
> 
> Do you have any other popular metrics to prove its popularity?
> 
> If not, you can return here once your project reaches 5000+ GitHub stars.

Alright, thanks, will be back after 5k Stars

**Comment by LitoMore** [comment]
I'm converting your PR to draft since it's out of scope.

Feel free to change it to ready once it reaches our requirements.

**Pull request closed without merging**